src/test/java/com/conveyal/osmlib/VanillaExtractTest.java

```diff
@@ -0,0 +1,39 @@
+package com.conveyal.osmlib;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class VanillaExtractTest {
+    static final String TEST_FILE = "./src/test/resources/porto_portugal.osm.pbf";
```

Comment:
Resources are associated with a package (in this case com.conveyal.osmlib) so it should be possible to put them in the appropriate place (/src/test/resources/com/conveyal/osmlib/porto_portugal.osm.pbf) then open them as a stream without giving full paths. That would require us to have load methods that accepted something other than a string path though.